src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,38 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  selectedTodo: Todo,
+  changeSelectedTodo: (value: null) => void,
+};
+
+export const TodoModal: React.FC<Props> = ({
+  selectedTodo,
+  changeSelectedTodo,
+}) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const {
+    id, title, userId, completed,
+  } = selectedTodo;
+
+  useEffect(() => {
+    setIsLoading(true);
+    getUser(userId)
+      .then((foundUser) => setUser(foundUser))
+      .catch()
```

Comment:
The same comment as above, you can print something to the console, or if you want to show some message on UI you can create `isError` state and show this message depending on this state value